samples/bluetooth/peripheral/src/main.c

```diff
@@ -337,19 +337,34 @@ static void bas_notify(void)
 	bt_bas_set_battery_level(battery_level);
 }
 
+static uint8_t bt_heartrate = 90U;
+
 static void hrs_notify(void)
 {
-	static uint8_t heartrate = 90U;
-
 	/* Heartrate measurements simulation */
-	heartrate++;
-	if (heartrate == 160U) {
-		heartrate = 90U;
+	bt_heartrate++;
+	if (bt_heartrate == 160U) {
+		bt_heartrate = 90U;
+	}
+
+	bt_hrs_notify(bt_heartrate);
+}
+
+int bt_hrs_ctrl_point_write(uint8_t request)
+{
+	printk("HRS Control point request: %d\n", request);
+	if (request != BT_HRS_CONTROL_POINT_RESET_REQ) {
+		return -ENOTSUP;
```

Comment:
And this illustrates the issue mentioned in https://github.com/zephyrproject-rtos/zephyr/pull/77598/files#r1732584303

If there are multiple listeners to this callback and this application does not support opcode X but another does, then the other application will never get the request.

Reply:
Yup i realized it while implementing the example code. I have planned to use -ENOTSUP as a pass thru error,
Which will continue handling of this requests with other listeners.